### PR TITLE
[jaeger-operator] Synchronize cluster role template with jaeger operator

### PR DESCRIPTION
**Pull request by majidazimi**
Cluster role template wasn't reflecting the same level of permissions as upstream jaeger-operator. This PR makes helm chart follow upstream.

The issue is discussed here: https://github.com/jaegertracing/helm-charts/issues/250

### CODE REVIEW

**Comment by SCLogo** [comment]
can we merge this fix ?

**Comment by jpkrohling** [comment]
Sorry for not looking into this earlier, I was waiting for a maintainer of the Helm chart to take a look first. In any case, I approved the run. Could you please sort out the DCO check?

**Comment by majidazimi** [comment]
I bumped chart version to `2.22.0` to pass the linter. Not sure whether that's the correct version number to merge.

**Review by jpkrohling — APPROVED** [review]
@cpanato , @batazor , would you like to review as well? I'll merge it tomorrow unless you express concerns.

**Comment by jpkrohling** [comment]
@majidazimi, I was about to merge this, but realized that you haven't signed your commit and this repository requires signing (!!). Are you able to sign it?

image


https://docs.github.com/articles/about-gpg/

**Comment by majidazimi** [comment]
ok signed.

**Pull request merged**
